forge-gui/res/cardsfolder/e/eldritch_pact.txt

```diff
@@ -0,0 +1,7 @@
+Name:Eldritch Pact
+ManaCost:6 B
+Types:Sorcery
+A:SP$ Draw | Cost$ 6 B | NumCards$ X | ValidTgts$ Player | TgtPrompt$ Choose a player | SubAbility$ DBLoseLife | SpellDescription$ Target player draws X cards and loses X life, where X is the number of cards in their graveyard
```

Comment:
`Cost$` is unneeded as long as it matches the ManaCost line above
`TgtPrompt$` is unneeded for most one word `ValidTgts$` such as this one
If it was a more complicated ValidTgts$ and you needed TgtPrompt$, a good rule of thumb is to make sure the word "target" is in there somewhere when we are targeting
Make sure your SpellDescription ends in `.`